src/Icons/doc/index.rst

```diff
@@ -526,13 +526,13 @@ TwigComponent
 ~~~~~~~~~~~~~
 
 The ``ux_icon()`` function is optimized to be as fast as possible. To deliver the
-same level of performance when using the HTML syntax (``<twig:ux:icon name="..." />``),
+same level of performance as if using the HTML syntax (``<twig:ux:icon name="..." />``),
```

Comment:
No, the original meaning is the current one. The HTML syntax is the name of the syntax of TwigComponents that looks like HTML. 

So here it's not "as if" but "when"

Reply:
What you want to say is:
> The twig syntax has the same performance as the HTML syntax.

Right?

Same as in https://github.com/symfony/ux/pull/2781#pullrequestreview-2876147891 - let's switch the parts:

> The TwigComponent overhead is reduced to deliver the same level of performance when using the HTML syntax

=> No. Cause when I use the twig syntax, I cannot use the HTML syntax at the same time.

Reply:
Not at all.

What we do say here is : using the Twig function is the fastest possible. When using the Twig Component with the HTML Syntax, we have developed tactics.

Said in another way: 

"To deliver the same level of performance when using the HTML syntax... the UX Icon component [...]"

Reply:
OK, I finally got what you mean. But this is impossible to get across with the current sentence structure. I'll come up with a better suggestion.
But one question first: Is `ux_icon()` now faster or are they the same?

Reply:
> But one question first: Is ux_icon() now faster or are they the same?

If they are in fact, the same, what about removing this paragraph entirely?

Reply:
> what about removing this paragraph entirely?

I didn't dare to ask that :-)

Reply:
But let's keep this PR focused on just the link fix and change this in #2795.

Reply:
> know how the sausage is made

A Canadian saying? :-) But you don't eat that much sausage over there, do you?

OK, I reverted it here. But that other PR is a mess already, so I won't add more stuff there...